### PR TITLE
Added dynamic partial import

### PR DESCRIPTION
**Pull request by loic294**
# Description

**Make sure the PR is against the `develop` branch (see [Contributing](https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md)).**

Please include a summary of the change(s) and which issue(s) it addresses. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ScoutSuite/frontend/src/pages/FlaggedItems/SelectedItemContainer/index.js [hunk 1]
```diff
@@ -1,31 +1,49 @@
...
-    children,
+    title
   } = props;
 
+  const path = (new URL(document.location)).searchParams.get('path');
+  const params = useParams();
+  const { data: finding, loading: l1 } = useAPI(`raw/services/${params.service}/findings/${params.finding}`);
+  const { data, loading: l2 }= useAPI(getItem(params.service, params.finding, params.item, path));
+
+  if (l1 || l2 || !data) return null;
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/pages/FlaggedItems/SelectedItemContainer/index.js:24]: c'est quoi le 11, 12?

Comment (loic294) [reply ScoutSuite/frontend/src/pages/FlaggedItems/SelectedItemContainer/index.js:24]: C'est 'loading1' et 'loading2' comme j'ai deux useAPI

File: ScoutSuite/frontend/src/partials/aws/s3.buckets.id/BucketInformations/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+import './style.scss';
+
+const propTypes = {
+  data: PropTypes.objectOf({
+    arn: PropTypes.string,
+    region: PropTypes.string,
+    CreationDate: PropTypes.string,
+    logging: PropTypes.bool,
+    defaultEncryption: PropTypes.bool,
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/partials/aws/s3.buckets.id/BucketInformations/index.js]: you forgot to also change the variables' name in propTypes.

File: ScoutSuite/frontend/src/partials/aws/s3.buckets.id/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+const Bucket = props => {
+  const {
+    data
+  } = props;
+
+  if (!data) return null;
+
+  console.log('DATA', data);
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/partials/aws/s3.buckets.id/index.js]: forgot the console.log

**Review by SamuelMeilleur — CHANGES_REQUESTED** [review]

**Review by SamuelMeilleur — APPROVED** [review]

**Pull request merged**